third_party/terraform/utils/provider_test.go.erb

```diff
@@ -60,10 +64,22 @@ var billingAccountEnvVars = []string{
 func init() {
 	testAccProvider = Provider().(*schema.Provider)
 	testAccRandomProvider = random.Provider().(*schema.Provider)
+	<% if version == 'ga' -%>
 	testAccProviders = map[string]terraform.ResourceProvider{
 		"google": testAccProvider,
 		"random": testAccRandomProvider,
 	}
+	<% else -%>
+	testAccProviders = map[string]terraform.ResourceProvider{
+		"google": testAccProvider,
```

Comment:
It feels like `testAccProviders` could be configured based on what version instead of using the OICS variable here. I think it would future proof for alpha/eap as well.
```suggestion
		"google<%= version == 'ga' ? "" : "-" + version  %>": testAccProvider,
```

Reply:
See https://github.com/GoogleCloudPlatform/magic-modules/pull/1464/files#diff-837b2eb2b73f9cf0e8561a61e60ed1ffR82 for (part) of why this is infeasible. Until we upstream some changes to the Terraform test gauntlet we can't import aliased resources, and we'd need to guarantee that every resource uses the alias.

That means every test file would need to be a `go.erb` to template that out (and doing that alone would add 5-10s to MM generation times!)